lang/ilias_de.lang

```diff
@@ -15943,3 +15943,8 @@ iass#:#iass_event_time_place_required#:#Verpflichtende Angabe von Datum und Ort
 iass#:#iass_event_time_place_required_info#:#In der Dokumentation zu einem Teilnehmer müssen Datum und Ort angegeben werden.
 iass#:#iass_place#:#Ort der Durchführung
 iass#:#iass_event_time#:#Datum der Durchführung
+sess#:#preview_crs_title#:#Beispielkurs
+sess#:#preview_crs_link#:#http://localhost/cat_ilias/tms/goto.php?target=crs_&client_id=cattms
```

Comment:
Vielleicht wirklich durch etwas Neutraleres ersetzen? 
https://concepts-and-training.de/tms/login.php?target=&client_id=cattms
??? keine Ahnung.